guide/installation-and-configuration/pre-installation-checklist/verify-signatures.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: default
+title: Verifying package signatures
+published: true
+sorting: 40
+tags: [getting started, installation]
+---
+
+On the [Download CFEngine][d], you will find sha256 checksums of all downloadable
+files which you can verify by using `sha256sum` tool.
+
+[d]: http://cfengine.com/product/free-download
+
+In addition to this, `*.deb` and `*.rpm` packages (with except of AIX rpm) are
+cryptographically signed using gpg.
+
+
+## Validating signature of RPM
+
+NOTE: The AIX rpm is currently NOT signed because it's not supported on older versions of AIX.
+
+1. Import the public GPG key.
+
+    ```console
+    # rpm --import https://cfengine-package-repos.s3.amazonaws.com/pub/gpg.key
+    ```
+
+2. Validate the signature.
+
+    ```console
+    # rpm -K ./cfengine-nova-hub-3.12.2-2.x86_64.rpm 
+    ./cfengine-nova-hub-3.12.2-2.x86_64.rpm: rsa sha1 (md5) pgp md5 OK
+    ```
+
+    NOTE: If you don't import the public key first, you will get an error about the key missing:
+
+    ```console
+    # rpm -K ./cfengine-nova-hub-3.12.2-2.x86_64.rpm 
+    ./cfengine-nova-hub-3.12.2-2.x86_64.rpm: RSA sha1 ((MD5) PGP) md5 NOT OK (MISSING KEYS: (MD5) PGP#a86e7afa) 
+    ```
+
+## Validating signature of DEB
+
+1. Import the public GPG key.
+
+    ```console
+    # wget https://cfengine-package-repos.s3.amazonaws.com/pub/gpg.key
+    # mkdir /usr/share/debsig/keyrings/7061B663A86E7AFA
```

Comment:
mmm i see this key fingerprint in a few places, perhaps it should be a variable that is expanded?

e.g https://github.com/cfengine/documentation-generator/blob/master/_config.yml#L22 is then expanded in index.markdown:

```
index.markdown:Welcome! This is the documentation for CFEngine {{site.cfengine.core_branch}}.

```